### PR TITLE
AKS Secure Baseline RA - Using native CSI secrets store driver for key vault

### PR DESCRIPTION
**Pull request by magrande**

### CODE REVIEW

**Review by v-fearam — APPROVED** [review]

File: 05-aks-cluster.md [hunk 1]
```diff
@@ -23,6 +23,24 @@ Now that the [hub-spoke network is provisioned](./04-networking.md), the next st
    RESOURCEID_VNET_CLUSTERSPOKE=$(az deployment group show -g rg-enterprise-networking-spokes -n spoke-BU0001A0008 --query properties.outputs.clusterVnetResourceId.value -o tsv)
    ```
 
+1. Enable the Azure Key Vault Secrets Provider feature
```
Comment (ckittel) [inline 05-aks-cluster.md:26]: Typically we like to do preview feature enablement like this in the pre-req section of the docs. We have a step that comes and goes if we are using a preview feature. Check the file history for the style of that. (Typical we link to the instructions instead of duplicating them)

File: README.md [hunk 2]
```diff
@@ -40,7 +40,6 @@ Finally, this implementation uses the [ASP.NET Core Docker sample web app](https
 - [Flux GitOps Operator](https://fluxcd.io)
 - [Traefik Ingress Controller](https://doc.traefik.io/traefik/v2.4/routing/providers/kubernetes-ingress/)
 - [Azure AD Pod Identity](https://github.com/Azure/aad-pod-identity)
-- [Azure KeyVault Secret Store CSI Provider](https://github.com/Azure/secrets-store-csi-driver-provider-azure)
```
Comment (ckittel) [inline README.md:43]: Why is this removed? We still use it. Can you instead link to the docs page? While you're here, can you do the same for the line above?

**Review by ckittel — CHANGES_REQUESTED** [review]
Couple small changes request. If you have any questions, just ping me today. Thanks

File: 05-aks-cluster.md (hunk 1 above)
Comment (ckittel) [reply 05-aks-cluster.md:26]: The format is the following.

```markdown
1. While the following feature(s) are still in _preview_, please enable them in your subscription.
   1. [Register the Azure Policy for AKS preview feature - `AKS-AzureKeyVaultSecretsProvider`](https://docs.microsoft.com/azure/...).
```

And we inject that after the Azure CLI install step

**Review by ckittel — APPROVED** [review]
:)
